src/main/kotlin/com/sourcegraph/cody/history/state/LLMState.kt

```diff
@@ -7,7 +7,7 @@ import com.sourcegraph.cody.agent.protocol.ChatModelsResponse
 
 @Tag("llm")
 class LLMState : BaseState() {
-  @get:OptionTag(tag = "model", nameAttribute = "") var model: String? by string()
```

Comment:
are we going to keep intellij's persistance? or is it to be handled by the agent? if we want to keep it we would need a migration to migrate the old state

Reply:
Oh, I see. Then, I'd keep this part unchanged. Let me push an update.